spec/manageiq_performance/configuration_spec.rb

```diff
@@ -207,90 +195,90 @@
     end
 
     it "defines ManageIQPerformance.config.default_dir" do
-      expect(ManageIQPerformance.config.default_dir).to eq("/tmp/miq_perf")
-      expect(ManageIQPerformance.config["default_dir"]).to eq("/tmp/miq_perf")
+      expect(ManageIQPerformance.config.default_dir).to eq "/tmp/miq_perf"
+      expect(ManageIQPerformance.config["default_dir"]).to eq "/tmp/miq_perf"
     end
 
     it "defines ManageIQPerformance.config.log_dir" do
-      expect(ManageIQPerformance.config.log_dir).to eq("tmp/my_log_dir")
-      expect(ManageIQPerformance.config["log_dir"]).to eq("tmp/my_log_dir")
+      expect(ManageIQPerformance.config.log_dir).to eq "tmp/my_log_dir"
+      expect(ManageIQPerformance.config["log_dir"]).to eq "tmp/my_log_dir"
```

Comment:
@kbrock This is actually from https://github.com/ManageIQ/manageiq-performance/pull/28, which makes much more sense being there.  A better diff to look at would be this:

https://github.com/ManageIQ/manageiq-performance/compare/global_config_changing_interfaces...memory_middleware_storage